internal/driver/commands.go

```diff
@@ -273,7 +289,11 @@ func usage(commandLine bool) string {
 	}
 
 	help = help + strings.Join(commands, "\n") + "\n\n" +
-		"  Options:\n"
+		"  Options:\n" +
+		"    General format is <option>=<value>\n" +
+		"    <f> is a float, <n> is an integer, and <s> is a string\n" +
+		"    If the option is a boolean value, then you can simply type <option>\n" +
+		"    Type \"help <option>\" for detailed usage\n\n"
```

Comment:
Sorry to change my mind on this.  I'm going to suggest removing this from the help output.

Describing `<s>`, `<f>`, and `<n>` for each option should be sufficient, and its hard to precisely describe usage here (particularly for boolean options, and setting boolean options to false).

Reply:
So I should just keep line 292 through 294, right?

Reply:
Yes. Removing 295/296 and keeping 292-294 will work.

Reply:
Done.